alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -278,8 +278,6 @@ class AlertIndices(
         val alertsIndex = dataSources.alertsIndex
         if (!clusterService.state().routingTable().hasIndex(alertsIndex)) {
             alertIndexInitialized = createIndex(alertsIndex!!, alertMapping())
-        } else {
-            updateIndexMapping(alertsIndex!!, alertMapping())
```

Comment:
Updating the mappings on each monitor execution causes a perf degradation. I am not sure if remove this is safe or if another solution is needed. Will look into it soon